### DISCUSSION TITLE
媒体库希望增加飞牛影视的媒体库

### DISCUSSION THREAD

**Discussion by jinws1993** [jxxghp/MoviePilot#3319] [建议]
希望作者大大增加一个飞牛影视的媒体库，最近把emby和jellyfin都弃用了，打算只保留一个飞牛影视库。

**Comment by bwliangc** [comment]
赞同

**Comment by rongshuizhou** [comment]
赞同+1

**Comment by stkevintan** [comment]
但是目前好像飞牛没有提供公开API, 建议围观 https://club.fnnas.com/forum.php?mod=viewthread&tid=15472

**Reply by cddjr** [reply]
mp已经加添加支持了

**Reply by stkevintan** [reply]
@cddjr 求助
请检查服务端地址 http://192.168.xx.xx:8005
api.py 请求接口 http://192.168.xx.xx:8005/api/v1/sys/version 失败
api.py请求接口 http://192.168.xx.xx:8005/v/api/v1/sys/version 失败

**Reply by cddjr** [reply]
飞牛默认端口5666，你怎么配置8005

**Reply by stkevintan** [reply]
飞牛nas是5666但是飞牛影音app上显示的是8005 另外
请检查服务端地址 http://192.168.xx.xx:5666
api.py 请求接口 http://192.168.xx.xx:5666/api/v1/sys/version 失败
api.py 请求接口 http://192.168.xx.xx:5666/v/api/v1/sys/version 失败

另外我在本地都能通过
http://192.168.xx.xx:8005
http://192.168.xx.xx:5666/v
访问web版飞牛影视

**Reply by cddjr** [reply]
那就是mp连不上这个ip地址，检查容器之间的连通性、检查是否有防火墙规则。

**Reply by stkevintan** [reply]
想起来了 mp在macvlan里 谢谢啦
